app/assets/javascripts/directives/is_float.js

```diff
@@ -0,0 +1,21 @@
+ManageIQ.angular.app.directive('isFloat', function() {
```

Comment:
This is a duplicate of angular validator's `is-float`.

So I think you can just remove this file and it will work equally well ;)